src/index.ts

```diff
@@ -2,12 +2,15 @@ import { createElement } from 'react';
 import { createRoot } from 'react-dom/client';
 import ReactModal from 'react-modal';
 import { Logger } from 'ethers/lib.esm/utils';
+import Decimal from 'decimal.js';
 
 import './styles/main.css';
 
 import Entry from './Entry';
 import store from '~redux/createReduxStore';
 
+Decimal.set({ toExpPos: 78 });
```

Comment:
This is taken from the dapp and stops the Decimal class from trunating large numbers with exponent notiation (up to but not incl the power of 78) https://mikemcl.github.io/decimal.js/#toExpPos